File: modules/old-algorithms/modules/algorithms-implementations/recursion/src/cpp/driver_sumElements_iterative_approach.cpp
#include <iostream>
#include <vector>

int sumElements(std::vector<int> list)
{
    int sum = 0;
    for (int element : list)
        sum += element;
    return sum;
}

int main()
{
    std::vector<int> myList = {1, 3, 5, 7, 9};
    std::cout << "The sum of all element is: " << sumElements(myList);
    return 0;
}

File: modules/old-studies/old-algorithms-and-ds/modules/algorithms-implementations/recursion/src/python/driver_sumElements_iterative_approach.py
def sumElements(myList: list[int]) -> int:
    sum = 0
    for element in myList:
        sum += element
    return sum


if __name__ == "__main__":
    myList: list[int] = [1, 3, 5, 7, 9]
    print("The sum of all element is:", sumElements(myList))
